bin/markcop

```diff
@@ -288,11 +288,21 @@ function spellcheck {
 # $contents
 for check in "${CHECKS[@]}"; do
   printf "\n\nRunning check: $check...\n"
-  for file in $(git ls-files | grep '\.md$'); do
-    content=$(grep -n "" $file | perl -0777 -pe 's/[0-9]+\:```.*?```//gs')
-    # $content is in the format file:line_number:line
-    $check "$file" "$content"
-  done
+  # If there are no arguments, check all markdown files in the current repo
+  if [ -z "$1" ]; then
```

Comment:
`if [ -z "$1" ]` checks if the length of the string `"$1"` is zero. `$#` returns the number of arguments, perhaps a check with it would be better? Ex. `if [ "$#" -e 0 ]`.